package.json

```diff
@@ -43,10 +43,11 @@
     "lint": "run-s lint:check prettier:check",
     "lint:check": "eslint . --ext .ts,.tsx,.js,.jsx",
     "lint:fix": "pnpm lint:check --fix",
-    "node:dev": "fuelhat node",
-    "nodes:start": "fuelhat node start -d && fuelhat node start -t -d",
-    "nodes:stop": "fuelhat node stop -d && fuelhat node stop -t -d",
-    "nodes:reset": "pnpm nodes:stop && pnpm nodes:start",
+    "node:dev": "make -C ./docker start",
+    "node:test": "make -C ./docker start-test",
+    "nodes:start": "run-s node:dev node:test",
+    "nodes:stop": "make -C ./docker stop",
+    "nodes:reset": "run-s nodes:stop nodes:start",
```

Comment:
can we change to:

`node:up` 
`node:test:up` 
`node:down`
`node:test:down`
`node:clean`
`node:test:clean`


`nodes:up`
`nodes:down`
`nodes:clean`
`nodes:reset`